### ISSUE TITLE
[Feature Request] DirectX Renderer 

### ISSUE THREAD

**Issue by powellnorma** [AllenDang/giu#731]
### Related problem

https://github.com/AllenDang/giu/issues/170#issue-863120297
https://github.com/AllenDang/giu/issues/147#issuecomment-789963826

### Your request

I'd be useful to be able to use the DirectX renderer, so that the GUI starts even if no "real" GPU driver is installed. I tested the demos from https://github.com/ocornut/imgui#demo and the directX executables work just fine. Is this feasible?

**Comment by powellnorma** [comment]
Ok, what I found was this:
https://github.com/AllenDang/giu/blob/04cd961c1d12d31a7bcd329350a686d6c76cc3fe/MasterWindow.go#L88

So probably some wrapper logic would have to be implemented in imgui-go first?
https://github.com/AllenDang/cimgui-go/blob/f9856eb4a9a725c37e34b415d3eabc00e90276fa/glfw_backend.go#L187

**Comment by gucio321** [comment]
Yep. There is a backend implementation in cimgui and other backend should be implemented there first. Unfortunately i can't do anything about directx as I'm a Linux user
